Userland/Utilities/shot.cpp

```diff
@@ -165,6 +170,13 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
+    if (edit_image) {
+        char const* argv[] = { "PixelPaint", output_path.characters() };
+        pid_t child_pid;
+        if ((errno = posix_spawn(&child_pid, "/bin/PixelPaint", nullptr, nullptr, const_cast<char**>(argv), environ)))
+            perror("posix_spawn");
```

Comment:
This is one line with the wrappers in LibCore :^)
```suggestion
        TRY(Core::Process::spawn("/bin/PixelPaint", Array { output_path });
```

Reply:
(You will need to `#include <LibCore/Process.h>`)

Reply:
Also, I just spotted your `argv` is non-nullptr terminated which could lead to a bad time... Another reason why you should prefer the wrappers.

Reply:
Thanks for the feedback! I'm, new to the codebase so I was not aware of that macro.

Changes have been addressed.